test/testground/compositions/standard/plan.toml

```diff
@@ -1,39 +1,40 @@
 [metadata]
-name = "standard"
 author = "core-app"
 
 [global]
-plan = "celestia"
+plan = "core-app"
```

Comment:
[Question] Does this change require updating the test instructions [here](https://github.com/celestiaorg/celestia-app/tree/main/test/testground#running-the-experiment) where the plan name is `celestia`?

Reply:
idk, as I think I've done both and it worked, but good point and changed here 9ba189188e4711af942257bccf60d59cd301001c